_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -279,7 +279,9 @@ public static function route( $path, $classname, $method,
 	 */
 	public static function dismiss_notice( $data ) {
 		$notice = $data['notice'];
-		if ( isset( $notice ) && ! empty( $notice ) ) {
+		$param = $data->get_json_params();
+
+		if ( isset( $param['dismissed'] ) && $param['dismissed'] === true && isset( $notice ) && ! empty( $notice ) ) {
```

Comment:
Thanks. I was waiting for the review because I thought that if we throw a new error right now, we would need a new translation for that new error message. And was thinking in adding an new issue to handle the new error (this applies to three of the endpoints. What do you think?